packages/schematics/src/collection/ngrx/rules/add-imports-to-module.ts

```diff
@@ -1,5 +1,5 @@
 import { Rule, Tree } from '@angular-devkit/schematics';
-import { Change } from '@ngrx/schematics/src/utility/change';
+import { Change } from '@ngrx/schematics/schematics-core';
```

Comment:
I'd not use this type.

It's used here.

```
  const addImport = (symbolName: string, fileName: string): Change => {
      return insertImport(source, modulePath, symbolName, fileName);
    };
```

insertImport is provided by Angular schematics, so it seems wrong to use a type provided by NgRx for a function call provided by Angular

Reply:
Fixed with `import { Change } from '@schematics/angular/utility/change';`